Documentation-Review-Process.md

```diff
@@ -0,0 +1,164 @@
+# Documentation review process
+
+* [Introduction](#introduction)
+* [The Documentation Team](#the-documentation-team)
+* [Default document review process](#default-document-review-process)
+* [Justification for different process](#justification-for-different-process)
+    * [Users should always expect accurate Documentation](#users-should-always-expect-accurate-documentation)
+    * [Some documentation PRs can be fast-tracked](#some-documentation-prs-can-be-fast-tracked)
+    * [Formal doc reviews can be very time-intensive](#formal-doc-reviews-can-be-very-time-intensive)
+* [Decision Authority](#decision-authority)
+* [Determine if a full document review is required](#determine-if-a-full-document-review-is-required)
+    * [Exempt](#exempt)
+    * [Required](#required)
+* [Requesting a formal document review](#requesting-a-formal-document-review)
+* [Lightweight document review process](#lightweight-document-review-process)
+
+## Introduction
+
+The process for reviewing documentation PRs is slightly different from
+[the general process for reviewing code PRs](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#reviewsPR-Review-Guide.md).
+This document explains the differences.
+
+## The Documentation Team
+
+The Documentation Team are responsible for carrying out a full documentation
```

Comment:
are/is: The Documentation Team is responsible for carrying out a full documentation